exasol/analytics/query_handler/graph/stage/sql/dependency.py

```diff
@@ -11,16 +11,17 @@
 @dataclasses.dataclass(frozen=True)
 class Dependency:
     """
-    This class represents that a object depends on something which in fact can depend on something else.
-    That exactly this means is user defined.
-    For example, this could represent that a view depends on a certain table.
+    An instance of this class expresses that a object depends on something
+    which in fact can depend on something else.  The exact meaning of a
+    dependency is user-defined.  For example, a dependency could express that
+    a view depends on a certain table.
     """
```

Comment:
This is definitely a better English, but the phrase "object depends on something which in fact can depend on something else" still sounds a bit mystical. An instance of this class represents a node in the dependency graph, right? That's how I would describe it.

Reply:
yeah, that is probably a better wording. Had first to think, if it is really a graph or if it is only a tree, but yes it can be a graph.

Reply:
> sounds a bit mystical

I definitely agree :slightly_smiling_face: 
Please see updated PR.